container/cloudbuild.yaml

```diff
@@ -19,27 +19,17 @@
 timeout: 21600s
 
 steps:
-# Step: build a custom container with bazel
-  - name: 'gcr.io/cloud-builders/bazel'
-    args:
-    - '--output_base=/workspace'
-    - 'run'
-    - '${_BAZEL_TOOLCHAINS_REF}//container/ubuntu16_04/builders/bazel_docker_gcloud:bazel_${_BAZEL_VERSION}_docker_gcloud'
-    id: 'builder'
-    waitFor:
-    - "-" # wait for nothing - start immediately
-
 # Step: print the bazel version for confirmation
-  - name: 'bazel/container/ubuntu16_04/builders/bazel_docker_gcloud:bazel_${_BAZEL_VERSION}_docker_gcloud'
+  - name: 'gcr.io/asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud:${_BAZEL_VERSION}'
```

Comment:
this would mean that effectively the argo build can only be executed from the asci-toolchains projects to have access to the private container, right?

Reply:
Yes

Reply:
Ok, lets change https://github.com/bazelbuild/bazel-toolchains/blob/master/container/README.md to make explicit this process can currently only be executed if by users who have access to asci-toolchains gcr but we publish it here to make the build process for our containers publicly available. Also add pointers here to how we build each of these containers (e.g., just pointing to the target in this repo we used to build the container should be sufficient?)

Reply:
Updated. Didn't list individual targets, but pointing users to look at build locally section instead.